.github/workflows/pr.yaml

```diff
@@ -51,12 +51,15 @@ jobs:
 
     - name: Deploy the chart
       run: |
-        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace default
+        kubectl create namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
+        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
         # Sadly --wait fails, so have to sleep, look to maybe pull in mock test to solve
         # this problem another way
         sleep 60
     - name: Run the tests
-      run: helm test $GITHUB_HEAD_REF-$SHORT_SHA
+      run: helm test $GITHUB_HEAD_REF-$SHORT_SHA --namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
```

Comment:
If we run at the same time PR check and branch, or tag based on the same git commit?
is it have sense to replace chart name from `$GITHUB_HEAD_REF-$SHORT_SHA` to `$ACTOR_LOWERCASE-$GITHUB_RUN_ID` for example?

Reply:
`$GITHUB_RUN_ID ` is used in the namespace if you need it for identification.

Reply:
I need to avoid error when we have concurrent jobs with same chart names